### PR TITLE
Commandline output includes all commands needed to create kubeconfig with auth info

### PR DESCRIPTION
**Pull request by jpweber**
Updated the commandline output so that a user that does not have an existing kubeconfig can take the commands and be able to connect to a cluster. I believe this resolves issued #4 

To achieve this I introduced including the kube-apiserver url to the config so that can be included in the output. Additionaly the public CA cert from is also being output because it is needed in the creation of a kube config. 
 
Readme and example have been updated to reflect these changes

### CODE REVIEW

**Comment by jpweber** [comment]
screen shot 2018-04-25 at 1 20 20 pm

Added screenshot to see what this new output looks like.

File: cmd/gangway/handlers.go [hunk 1]
```diff
@@ -155,6 +160,14 @@ func parseToken(idToken string) (*jwt.Token, error) {
 
 func commandlineHandler(w http.ResponseWriter, r *http.Request) {
 
+	// read in public ca.crt to output in commandline copy/paste commands
+	file, err := os.Open(clusterCAPath)
+	if err != nil {
+		log.Fatal(err)
```
Comment (craigtracey) [inline cmd/gangway/handlers.go]: Should we handle this better?

File: cmd/gangway/handlers.go [hunk 2]
```diff
@@ -30,6 +32,7 @@ import (
 
 const (
 	templatesBase = "templates"
+	clusterCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```
Comment (craigtracey) [inline cmd/gangway/handlers.go]: This forces gangway to be run in-cluster. There could be scenarios where this is undesirable and/or a different CA may want to be used.  Can we have a config option with this as the default?

**Comment by jpweber** [comment]
@craigtracey Changes have been made to resolve your requests.

**Pull request merged**